priv/repo/seeds.exs

```diff
@@ -26,7 +26,7 @@ alias Cadet.Accounts.{
 # Cadet.Repo.insert!(%Cadet.Settings.Sublanguage{chapter: 1, variant: "default"})
 
 if Cadet.Env.env() == :dev do
-  number_of_students = 100_000
+  number_of_students = 100
```

Comment:
Any reason this is reduced again?

Reply:
it was taking too long to reset the db :(

Reply:
Oof 😢 ok maybe we will see how then, we can change it back later haha

Reply:
Though I think we can put it to 1000 to mimic 1101S?

Reply:
yeah, 1000 would be good! doesnt take too long for a one time set up (at most a minute).